### PR TITLE
Failover RGW pods tests

### PR DESCRIPTION
**Pull request by ebenahar**
Signed-off-by: Elad Ben Aharon <ebenahar@redhat.com>

### CODE REVIEW

File: tests/manage/mcg/test_mcg_resources_disruptions.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                *['noobaa_core'], marks=pytest.mark.polarion_id("OCS-2232")
+            ),
+            pytest.param(
+                *['noobaa_db'], marks=pytest.mark.polarion_id("OCS-2233")
+            )
+        ]
+    )
+    def test_restart_noobaa_resources(self, resource_to_delete):
+        """
+        Test Noobaa resources restart and check Noobaa health
```
Comment (fbalak) [inline tests/manage/mcg/test_mcg_resources_disruptions.py:43]: NooBaa resources are not restarted but deleted. After they are deleted the test waits for a new one to be created but it is not the original one so it is not restarted in my opinion.

Comment (ebenahar) [reply tests/manage/mcg/test_mcg_resources_disruptions.py]: done

**Comment by ebenahar** [comment]
> Looks good but it would be good to add docstring comments for test steps to improve readability.

Added

**Comment by vavuthu** [comment]
LGTM.

Since we scale down RGW pods to 1 and 0, are we bringing back RGW pods to original state. If not will this test case doesn't effect the other test cases?

**Comment by ebenahar** [comment]
> LGTM.
> 
> Since we scale down RGW pods to 1 and 0, are we bringing back RGW pods to original state. If not will this test case doesn't effect the other test cases?

yes, here https://github.com/red-hat-storage/ocs-ci/pull/2730/files#diff-4d34189c6531b72ebec1e27d784bed8dR95-R97

**Comment by vavuthu** [comment]
> > LGTM.
> > Since we scale down RGW pods to 1 and 0, are we bringing back RGW pods to original state. If not will this test case doesn't effect the other test cases?
> 
> yes, here https://github.com/red-hat-storage/ocs-ci/pull/2730/files#diff-4d34189c6531b72ebec1e27d784bed8dR95-R97

Thanks. Missed that

**Pull request merged**